src/foam/nanos/notification/EmailSetting.js

```diff
@@ -91,13 +91,12 @@ foam.CLASS({
           if ( config != null ) {
             notification.getEmailArgs().put("link", config.getUrl());
           }
-        }    
+        }
 
         try {
           if ( foam.util.SafetyUtil.isEmpty(notification.getEmailName()) ) {
-            message.setSubject(notification.getTemplate());
             message.setBody(notification.getBody());
-            EmailsUtility.sendEmailFromTemplate(x, null, message, null, null);
+            EmailsUtility.sendEmailFromTemplate(x, null, message, "NOC", null);
```

Comment:
This is just plain wrong.  If an email template is not provided, then emails are not generated. 
You can generate a warning if you like.. 
so in this first part do not call sendEmailFromTemplate.